docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -154,8 +154,47 @@ The above conditions can be built into more complex expressions using the follow
   </tbody>
 </table>
 
+### Composite conditions
+
+Finally, there are a set of pre-built conditions which make it easier to construct common combinations of the above conditions.
+
+| Condition                                         | Description                                                                                            |
+| ------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `AutomationCondition.any_deps_updated`            | Returns true if any dependencies have been updated since the previous tick                             |
```

Comment:
Two comments
- For all the ones below, I would avoid `Returns true` language and instead focus on the outcome. So maybe `Executes if ...` or `Starts a run if ...` etc. 
- Avoiding the tick language if possible would be nice here too. Maybe something like `Materializes this asset if any of its dependencies have been updated since the last scheduled update`

Reply:
I avoided the "starts a run if" language because these conditions are generally not recommended as standalone policies to put on an asset. i.e. you really would not want to have `@asset(automation_condition=AutomationCondition.any_deps_missing())` (why would you want to materialize if any dependencies were missing?)

Maybe just removing the "returns true if" entirely is a bit clearer overall.

Would swapping "tick" to "evaluation" feel a bit better? In this case, it is fairly relevant that the value returned by these conditions is dependent on things that happen between discrete evaluations. Made that update.

Reply:
Ah that makes sense, yea these are composable. I think The update looks good!